modules/csharp/pages/new-logging-api.adoc

```diff
@@ -0,0 +1,18 @@
+= New Logging API
+:page-status:
+:page-edition: 
+:page-aliases: 
+ifdef::show_edition[:page-edition: {release}]
+ifdef::prerelease[:page-status: {prerelease}]
+:page-role:
+:description: Couchbase Lite 3.2.2 introduces a new Logging API.
+:keywords: edge mobile api java kotlin android apple logging file console custom 
+
+[abstract]
+{description}
+
+// Sets the correct mobile sdk for xref links in the common page
+include::partial$_set_page_context_for_csharp.adoc[]
+
+// ROOT:partial$commons/common-
+include::{root-commons}new-logging-api.adoc[]
```

Comment:
This partial contains Java style code snippets inline

Reply:
Okay, I _think_ these have all been dealt with by other fixes. Please take a look at [common-new-logging-api.adoc](https://github.com/couchbase/docs-couchbase-lite/blob/0463055e06862f8628b4f84a7ae83c6b9b0fc449/modules/ROOT/pages/_partials/commons/common-new-logging-api.adoc) and make sure I haven't missed anything.

Reply:
Looks good to me